<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConsensusSelectorDialog</class>
 <widget class="QDialog" name="ConsensusSelectorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>455</width>
    <height>275</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Consensus representation</string>
  </property>
  <property name="toolTip">
   <string>Select consensus representation scheme</string>
  </property>
  <property name="statusTip">
   <string>Select consensus representation scheme</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Consensus type</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="3">
      <widget class="QComboBox" name="consensusCombo">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="thresholdLabel">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Threshold</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSlider" name="thresholdSlider">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="tracking">
        <bool>true</bool>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::NoTicks</enum>
       </property>
       <property name="tickInterval">
        <number>10</number>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QSpinBox" name="thresholdSpin">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>0</height>
        </size>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::UpDownArrows</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QToolButton" name="thresholdDefaultButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Reset to default value</string>
       </property>
       <property name="text">
        <string>Reset</string>
       </property>
       <property name="icon">
        <iconset resource="../../../../U2Gui/U2Gui.qrc">
         <normaloff>:/core/images/arrow_rotate_clockwise.png</normaloff>:/core/images/arrow_rotate_clockwise.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextEdit" name="descriptionEdit">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../../U2Gui/U2Gui.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>ConsensusSelectorDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>338</x>
     <y>173</y>
    </hint>
    <hint type="destinationlabel">
     <x>192</x>
     <y>97</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>ConsensusSelectorDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>173</y>
    </hint>
    <hint type="destinationlabel">
     <x>192</x>
     <y>97</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
